modules/react/pagination/stories/pagination.stories.mdx

```diff
@@ -212,172 +119,142 @@ provide a configuration as in the example below.
 
-And for the hoisted model pattern, follow this table:
+If you're using the hoisted model pattern, follow this table instead:
 
-| name       | type              | required? | default | recommended                              |
-| ---------- | ----------------- | --------- | ------- | ---------------------------------------- |
-| aria-label | `string`          | ✅ `true` | n/a     | "Pagination" (and translated equivalent) |
-| model      | `PaginationModel` | ✅ `true` | n/a     | n/a                                      |
+<ArgsTable of={PaginationHoistedComponent} />
 
-This component also supports
-[all native HTMLElement props](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes).
-The `aria-label` prop is required for accessibility. We recommend using `"Pagination"` as seen in
-the example.
-
----
+Note that you must set `aria-label` in either case to meet accessibility standards. We recommend
+setting it to `Pagination` or the translated equivalent.
 
 ### Pagination.Controls
 
-`Pagination.Controls` is a styled `div` wrapper that provides proper alignment and spacing between
-elements inside `Pagination`. It does not handle any internal business logic or state.
-
 #### Usage
 
+`Pagination.Controls` is a styled `<div>` element that provides proper alignment and spacing between
+elements inside `Pagination`. It does not handle any internal business logic or state.
+
 ```tsx
-<Pagination.Controls>{/* child components go here */}</Pagination.Controls>
+<Pagination.Controls>{/* Child components */}</Pagination.Controls>
 ```
 
-#### Component Props
+#### Props
 
-This component supports
-[all native HTMLDivElement props](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div#Attributes).
+`Pagination.Controls` does not have any documented props.
 
----
+Undocumented props are spread to the underlying `<div>` element.
 
 ### Pagination.JumpToFirstButton
 
-`Pagination.JumpToFirstButton` is an `IconButton` that subscribes to `Pagination`'s context. This
-allows it to know when to disable and set `currentPage` to the first page.
-
 #### Usage
 
+`Pagination.JumpToFirstButton` is an `IconButton` that subscribes to the `Pagination` context. This
+allows it to know when to disable and set `currentPage` to the first page.
+
 ```tsx
 <Pagination.JumpToFirstButton aria-label="First" />
 ```
 
-#### Component Props
-
-| name       | type     | required? | default | recommended                         |
-| ---------- | -------- | --------- | ------- | ----------------------------------- |
-| aria-label | `string` | ✅ `true` | n/a     | "First" (and translated equivalent) |
+#### Props
 
-This component also supports all `IconButton` props.
+Undocumented props are spread to the underlying `<button>` element. Note that you must set
+`aria-label` to meet accessibility standards. We recommend setting it to `First` or the translated
+equivalent.
 
----
+`Pagination.JumpToFirstButton` supports all `IconButton` props.
```

Comment:
Rather than show an `<ArgsTable>` for `Pagination.JumpToFirstButton` and all of the other `IconButton` subcomponents (which would repeat the `IconButton` props several times throughout the doc), I just left it at this short sentence.